### ISSUE TITLE
Font with Stylistic Set(s)

### ISSUE THREAD

**Issue by KHoogteyling** [KevinGutowski/betterTypePanel#16]
Sorry if I post my question in wrong category…
My question is this: I use a Font 'Platform' with different Stylistic Sets and would like to use only 'Stylistic Set 1' (which has different style rounded 'a') because of Corporate Design Specifications — see attachment. 
Q: Will / could your plugin make this possible in Sketch 53? 
![schermafbeelding 2019-02-14 om 08 58 05](https://user-images.githubusercontent.com/28309797/52771801-2a265300-3037-11e9-8ca0-a59b0b74b5d1.png)

**Comment by KevinGutowski** [comment]
Yep! I’ll add that to the list of stretch goals. I don’t think that it will make it into initial release but it is something that I want to support!

**Comment by mtclmn** [comment]
100 votes for this please 💯 :)
